src/java.base/share/classes/sun/security/ec/SunEC.java

```diff
@@ -252,11 +252,6 @@ public Object newInstance(Object ctrParamObj)
                                 if (nativeCryptTrace) {
                                     System.err.println("EC KeyPair Generation - Not using OpenSSL integration due to older version of OpenSSL (<1.1.0).");
                                 }
-                            } else if (isAIX) {
```

Comment:
While you're here, please remove `isAIX` and use `OperatingSystem.isAix()` instead.

Reply:
Removed isAIX check that was here previously and and instead now use `OperatingSystem.isAix()` in this file.

Reply:
I don't see that change in 2705bd931da889b65b7f51149b4a321199272f1c.

Reply:
I made this update to `NativeECKeyPairGenerator.java` but not to `SunEC`. This class has also now been updated to use the `jdk.internal.util.OperatingSystem` class.